packages/react/src/components/Modal/Modal.mdx

```diff
@@ -1,13 +1,295 @@
+
+```javascript
+<Modal danger>
+  <ModalHeader />
+  <p className="bx--modal-content__text">The modal body content</p>
+</Modal>
+```
+
+With `<ComposedModal>`, you can control the buttons with the following code.
+
+| Button group variant                                                                                 | Usage                                                                                                                                                                                                                                                          |
+| ---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| No button ([passive modal](https://www.carbondesignsystem.com/components/modal/usage#passive-modal)) | Use `<ComposedModal>` without `<ModalFooter>`                                                                                                                                                                                                                  |
+| One button                                                                                           | Use `primaryButtonText` in `<ModalFooter>`                                                                                                                                                                                                                     |
+| Two buttons                                                                                          | Use `primaryButtonText` and `secondaryButtonText` in `<ModalFooter>`                                                                                                                                                                                           |
+| Three buttons                                                                                        | Put three buttons as children of `<ModalFooter>`, instead of using `primaryButtonText` or `secondaryButtonText`. Using this option requires style adjustment defined in application-level CSS for `<ModalFooter>`, e.g. `.bx--modal-footer { padding: 25%; }`. |
+| Two buttons with danger button                                                                       | Set `danger` prop in `<ModalFooter>` (like below)                                                                                                                                                                                                              |
+
+```javascript
+<ComposedModal>
+  ...
+  <ModalFooter danger primaryButtonText="OK" secondaryButtonText="Cancel" />
+</ComposedModal>
+```
+
+<InlineNotification kind="warning">
```

Comment:
Same as above, need to import the notification component or use markdown quotes to reinforce emphasis.

Reply:
added import to resolve the issue above as well